### PR TITLE
feat(2114,2076): api-catalog-backend implemented cmdb code support and status outage

### PR DESCRIPTION
**Pull request by akhilmhdh**
# Closes

1. 2114
2. 2076

# Explain the feature/fix

1. Implemented cmdb code support
2. Implemented outage component data pull
3. Fixed invalid rhat UUID in owner loader failing
4. Added invalid flag to show validation status on cron

## Does this PR introduce a breaking change

No

## Screenshots

<details>
<summary>View Screenshots</summary>

</details>

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one feature request or issue (no stragglers)?
- [ ] Does the change have appropriate unit tests?
- [ ] Did tests pass?
- [x] Did you update or add any necessary documentation (README.md, WHY.md, etc.)?
- [x] Was this feature demoed and the design review approved?

### CODE REVIEW

File: packages/api-catalog-service/src/config.ts [hunk 1]
```diff
@@ -9,6 +9,10 @@ export type Config = {
   apiGatewayURL: string;
   apiGatewayToken: string;
   nsEncryptionKey: string;
+  outageStatusURL: string;
+  rhatServiceURL: string;
+  rhatServieUsername: string;
```
Comment (riginoommen) [inline packages/api-catalog-service/src/config.ts:14]: rhatServiceNowUsername

File: packages/api-catalog-service/src/config.ts [hunk 2]
```diff
@@ -9,6 +9,10 @@ export type Config = {
...
+  rhatServiePassword: string;
```
Comment (riginoommen) [inline packages/api-catalog-service/src/config.ts:15]:
```suggestion
  rhatServiceNowPassword: string;
```

File: packages/api-catalog-service/src/config.ts [hunk 3]
```diff
@@ -36,5 +40,9 @@ export const setupConfig = (): Config => {
     redisURI: process.env.REDIS_URI,
     apiGatewayURL: process.env.API_GATEWAY_URL || '',
     apiGatewayToken: process.env.API_GATEWAY_TOKEN || '',
+    outageStatusURL: process.env.OUTAGE_STATUS_URL || '',
+    rhatServiceURL: process.env.RHAT_SERVICE_NOW_URL || '',
+    rhatServieUsername: process.env.RHAT_SERVICE_NOW_USERNAME || '',
+    rhatServiePassword: process.env.RHAT_SERVICE_NOW_PASSWORD || '',
```
Comment (riginoommen) [inline packages/api-catalog-service/src/config.ts:46]:
```suggestion
    rhatServiceNowPassword: process.env.RHAT_SERVICE_NOW_PASSWORD || '',
```

File: packages/api-catalog-service/src/config.ts (hunk 2 above)
Comment (akhilmhdh) [reply packages/api-catalog-service/src/config.ts]: Changed

File: packages/api-catalog-service/src/config.ts (hunk 1 above)
Comment (akhilmhdh) [reply packages/api-catalog-service/src/config.ts]: Changed

**Review by riginoommen — APPROVED** [review]

**Review by hybridx — APPROVED** [review]

**Pull request merged**